RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

```diff
@@ -1,104 +1,79 @@
 import FWCore.ParameterSet.Config as cms
 
-RecoHiEgammaFEVT = cms.PSet(
+# AOD content 
+RecoHiEgammaAOD = cms.PSet(
     outputCommands = cms.untracked.vstring(
-    'keep recoSuperClusters_*_*_*',
-    'keep recoCaloClusters_*_*_*',
-    'keep EcalRecHitsSorted_*_*_*',
-    'keep floatedmValueMap_*_*_*',
-    'keep recoPFCandidates_*_*_*',
-    "drop recoPFClusters_*_*_*",
-    "keep recoElectronSeeds_*_*_*",
-    "keep recoGsfElectrons_*_*_*",
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
-    'keep recoPhotons_gedPhotonsTmp_*_*',
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerGED_*_*',
-    'keep recoElectronSeeds_ecalDrivenElectronSeeds_*_*',
-    'keep recoTrackExtras_electronGsfTracks_*_*',
-    'keep EBSrFlagsSorted_ecalDigis__*',
-    'keep EESrFlagsSorted_ecalDigis__*'
-    )
-    )
+        'keep floatedmValueMap_*_*_*',
```

Comment:
If you keep all floatedmValueMap's already here, then there is no need to repeat it again in the RECO and FEVT part
By the way: don't you believe that the list that was there in your previous commit (the same which is now in the RECO event content here below) is inclusive enough?

Reply:
You are right. It's duplicate in FEVT.. 
Indeed, In the AOD event content, there are no `floatedmValueMap_hi*`,
however other `floatedmValueMap` exist as below:
```
        'keep floatedmValueMap_eidRobustLoose_*_*',
        'keep floatedmValueMap_eidRobustTight_*_*',
        'keep floatedmValueMap_eidRobustHighEnergy_*_*',
        'keep floatedmValueMap_eidLoose_*_*',
        'keep floatedmValueMap_eidTight_*_*',
        'keep floatedmValueMap_lowPtGsfElectronSeedValueMaps_*_*',
        'keep floatedmValueMap_lowPtGsfElectronID_*_*',
```
I've checked the HI AOD event content list by using only one hi workflow[1]. 
(other hi workflows 140.53, 158.0  don't have AOD outputs)
To be safe in case another floatedmValueMap_* may exist in AOD additionally,
I have chose to use `'keep floatedmValueMap_*_*_*',` instead of those 7 specific cases.

@perrotta Do you think it would be better to put those 7 specific cases on the RecoHiEgammaAOD or just use `'keep floatedmValueMap_*_*_*',` for the safe ?

[1] 140.56_RunHI2018+RunHI2018+RECOHID18+HARVESTDHI18 = step2.root(AOD)